lib/const.rb

```diff
@@ -88,6 +88,23 @@
 # If CURL environment variable exists use it in lieu of curl.
 CURL = ENV['CURL'] || 'curl'
 
+# set certificate file location for lib/downloader.rb
+SSL_CERT_FILE = unless ENV['SSL_CERT_FILE'].to_s.empty? or !File.exist?(ENV['SSL_CERT_FILE'])
+  ENV['SSL_CERT_FILE']
+elsif File.exist?("#{CREW_PREFIX}/etc/ssl/certs/ca-certificates.crt")
+  "#{CREW_PREFIX}/etc/ssl/certs/ca-certificates.crt"
+else
+  '/etc/ssl/certs/ca-certificates.crt'
+end
+
+SSL_CERT_DIR = if ! ENV['SSL_CERT_DIR'].to_s.empty? and Dir.exist?(ENV['SSL_CERT_DIR'])
+  ENV['SSL_CERT_DIR']
+elsif Dir.exist?("#{CREW_PREFIX}/etc/ssl/certs")
+  "#{CREW_PREFIX}/etc/ssl/certs"
+else
+  '/etc/ssl/certs'
+end
```

Comment:
@supechicken rubocop thinks this would work:
```
SSL_CERT_FILE = if ENV['SSL_CERT_FILE'].to_s.empty? || !File.exist?(ENV['SSL_CERT_FILE'])
                  ENV['SSL_CERT_FILE']
                elsif File.exist?("#{CREW_PREFIX}/etc/ssl/certs/ca-certificates.crt")
                  "#{CREW_PREFIX}/etc/ssl/certs/ca-certificates.crt"
                else
                  '/etc/ssl/certs/ca-certificates.crt'
                end
SSL_CERT_DIR = if ENV['SSL_CERT_DIR'].to_s.empty? && !Dir.exist?(ENV['SSL_CERT_DIR'])
                 ENV['SSL_CERT_DIR']
               elsif Dir.exist?("#{CREW_PREFIX}/etc/ssl/certs")
                 "#{CREW_PREFIX}/etc/ssl/certs"
               else
                 '/etc/ssl/certs'
               end
```

Reply:
This fixes the issue.